app/src/main/java/com/example/marvelapp/adapter/CharacterAdapter.kt

```diff
@@ -32,7 +31,7 @@ class CharacterAdapter(
 
     class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         private val binding = CardviewCharacterBinding.bind(itemView)
-        fun bind(character: Character, listener: CharacterListener) {
+        fun bind(character: com.example.domain.entity.Character, listener: CharacterListener) {
```

Comment:
add the correct import instead of writing the path here

Reply:
done